lib/protoc/version.rb

```diff
@@ -1,3 +1,6 @@
 module Protoc
-  VERSION = '2.6.1'
+  # The version of the protobuf library from which these protoc binaries were built
+  PROTOBUF_VERSION = '2.6.1'
+  # This gem's version
+  VERSION = PROTOBUF_VERSION + '.1'
```

Comment:
why not make the versions match? The original intent was that they match so you could just `gem install protobuf <version>` and get the version of protoc you needed